app/export_converters.go

```diff
@@ -131,3 +131,13 @@ func ImportReactionFromPost(reaction *model.Reaction) *ReactionImportData {
 		CreateAt:  &reaction.CreateAt,
 	}
 }
+
+func ImportCustomEmoji(emoji *model.EmojiForExport) *LineImportData {
```

Comment:
Rather than having the new `EmojiForExport` type, I'd prefer to see this function renamed `ImportLineFromEmoji(string *model.Emoji, fileName string)`, taking those two parameters instead of the additional struct.